core/Tracker/Model.php

```diff
@@ -375,9 +411,21 @@ private function visitFieldsToQuery($valuesToUpdate)
         return array($updateParts, $sqlBind);
     }
 
+    private function deleteDuplicateAction($newActionId)
```

Comment:
Only deletes one action, but I think this is a redundancy w/ Actions::deleteDuplicateActions. Anyone see a problem w/ using that class from within Tracker/Model?